### ISSUE TITLE
Can't load galaxies on new empty MISP

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#4198]
There are 2 issues here:
1) `galaxies.kill_chain_order` is not in the `INSTALL/MYSQL.sql` file
2) `galaxies.kill_chain_order` is defined as NOT NULL but without DEFAULT

This gives the following error upon the initial galaxies update:
```
2019-02-21 17:13:58 Error: [PDOException] SQLSTATE[HY000]: General error: 1364 Field 'kill_chain_order' doesn't have a default value
Request URL: /galaxies/update
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(472): PDOStatement->execute(Array)
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(438): DboSource->_execute('INSERT INTO `mi...', Array)
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(1092): DboSource->execute('INSERT INTO `mi...')
#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(1942): DboSource->create(Object(Galaxy), Array, Array)
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(1760): Model->_doSave(Array, Array)
#5 /var/www/MISP/app/Model/Galaxy.php(85): Model->save(Array)
#6 /var/www/MISP/app/Model/Galaxy.php(93): Galaxy->__load_galaxies(0)
#7 /var/www/MISP/app/Controller/GalaxiesController.php(40): Galaxy->update(0)
#8 [internal function]: GalaxiesController->update()
#9 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(491): ReflectionMethod->invokeArgs(Object(GalaxiesController), Array)
#10 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(GalaxiesController), Object(CakeRequest))
#12 /var/www/MISP/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
```

**Comment by RichieB2B** [comment]
I found a nasty work around for those on MySQL 5.7 that need to get past this:
```
SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'STRICT_TRANS_TABLES',''));
```

**Comment by iglocska** [comment]
That is indeed a nasty work around.

**Comment by iglocska** [comment]
Fixed in 3718a89b9a2ffa81c313de18c08b590c61a9e44d
